src/viam/sdk/tests/mocks/mock_motion.hpp

```diff
@@ -89,10 +91,11 @@ class MockMotion : public sdk::Motion {
     std::string peek_destination_frame;
     double peek_heading;
     bool peek_stop_plan_called = false;
-    std::vector<sdk::geo_obstacle> peek_obstacles;
+    std::vector<sdk::geo_geometry> peek_obstacles;
     std::vector<sdk::GeometryConfig> peek_map_obstacles;
     std::shared_ptr<constraints> peek_constraints;
     std::shared_ptr<sdk::motion_configuration> peek_motion_configuration;
+    std::vector<sdk::geo_geometry> peek_bounding_regions;
```

Comment:
We've added this as a field here but we don't seem to actually set it or peek it anywhere. We should follow the example of how `peek_obstacles` is used (i.e., set `peek_bounding_regions` in the `move_on_globe` request and then verify that it's been set in `test_motion.cpp`).